app/employee-app/backend/modules/database/db_functions.bal

```diff
@@ -78,3 +79,100 @@ public isolated function getEmployees(EmployeeFilter filters, int 'limit, int of
             lead: dbEmployeeInfo.lead == 1
         };
 }
+
+# Retrieve organizational details from HRIS.
+#
+# + employeeStatuses - List of employee statuses to consider
+# + return - Returns the list of business units
```

Comment:
"Returns" is redundant here, since this doc comment will appear under the **Return** section.
(Check similar occurrences)
```suggestion
# + return - The list of business units
```